static/css/around-map.css

```diff
@@ -6,3 +6,16 @@
 .mapboxgl-popup {
     z-index: 2;
 }
+
+button {
```

Comment:
Bewarn that this selector is so generic that it will inevitability interfere with other buttons on the page.